lib/devise-security.rb

```diff
@@ -1,6 +1,34 @@
 # frozen_string_literal: true
 
-require 'active_record'
+unless defined?(DEVISE_ORM)
+  DEVISE_ORM = (ENV["DEVISE_ORM"] || :active_record).to_sym
+  DEVISE_ORM = :mongoid
```

Comment:
This looks like it's always going to set the constant to `:mongoid`. What I think you want is `DEVISE_ORM ||= :mongoid`

Reply:
I'll remove that line. It's old code, when I was only testing mongoid ORM.